docs/reference.rst

```diff
@@ -0,0 +1,54 @@
+.. _reference_penaltymodel:
+
+Reference Documentation
+***********************
+
+.. currentmodule:: penaltymodel
+
+This package implements the generation and caching of :term:`penalty model`\ s.
+
+The main function for penalty models is:
```

Comment:
This will look a bit strange, a lead sentence with a class documented underneath. I'd put "... models is :func:`get_penalty_model`."